Resilio Connect API/JSMonitor/app.js

```diff
@@ -36,3 +36,41 @@ setTimeout(function() {console.log(enumerateAgents());}, 5000);
 
 console.log(findArrayDiff([1, 5, 17, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160], [1, 5, 17, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 165]));
 
+var prevList;
+var callCounter = 0;
```

Comment:
these 2 variables belong inside the class.  otherwise, you are still using global variables that we want to avoid.